presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -2658,6 +2659,19 @@ public FeaturesConfig setRewriteCrossJoinWithArrayContainsFilterToInnerJoin(bool
         return this;
     }
 
+    public boolean isRewriteCrossJoinWithArrayNotContainsFilterToAntiJoin()
+    {
+        return this.rewriteCrossJoinWithArrayNotContainsFilterToAntiJoin;
+    }
+
+    @Config("optimizer.rewrite-cross-join-with-array-not-contains-filter-to-anti-join")
+    @ConfigDescription("Enable optimization to rewrite cross join with array not contains filter to anti join")
+    public FeaturesConfig setRewriteCrossJoinWithArrayNotContainsFilterToAntiJoin(boolean rewriteCrossJoinWithArrayNotContainsFilterToAntiJoin)
+    {
+        this.rewriteCrossJoinWithArrayNotContainsFilterToAntiJoin = rewriteCrossJoinWithArrayNotContainsFilterToAntiJoin;
+        return this;
+    }
+
```

Comment:
I don't think this is needed. Let's just make it a session param as we can turn it off/on easily now

Reply:
that's what rebecca usually suggests. Quote from another PR: "Session properties should always be backed by configuration properties and the default value should be based on a corresponding configuration property."

Reply:
Yeah I just feel some of these opportunistic optimization don't need a feature flag and more importantly we now have (unlike before) configs to turn off session props cluster-wide so it feels unnecessary